### ISSUE TITLE
Regarding proof of origin

### ISSUE THREAD

**Issue by damooo** [Nanopublication/nanopub-discussions#1]
Currently, as nanopubs are published on central infrastructure,  There seems no way to ensure if the signer has authority to publish a publication with an uri of a given origin. As uri origin may be mistreated as prrof to trust as in the case of other web documents.

Currently one can easily publish a pub with an uri like `https://ieee.org/pubs/1` even though one doesn't have any thing to do with ieee. If uri origin has no meaning, then is it good to consider urns?

**Comment by tkuhn** [comment]
This is an interesting topic.

Using a third-party URI in a nanopublication is of course not per-se a breach of authority. URIs are supposed to be reused by others, after all. But I see your point that if I provide a definition, for example, of a third-party URI, I might do something that I don't have the authority to.

On the other hand, this can also be seen as a feature. For example, sometimes popular vocabularies are abandoned, and then it's good if we don't fully depend on the original creators for the continued maintenance of the vocabulary.

Just my two cents.

Maybe you'd like to participate in our next Nano Session on 18 April (https://nanopub.net/sessions) to explain and discuss this a bit more?

**Comment by damooo** [comment]
Indeed, it is a feature to talk about things whose identities are in other domain. As they are namespaced by graph iri, there will not be problem. We can only trust statements in trusted publications.

Issue is more about uris of nanopubs in nanopub-servers. Not about uris of subjects in nanopub assertions.

As defined in nanopub-server network [paper](http://arxiv.org/pdf/1411.2749), one can mint trusty iri of a nanopub with any random origin, and push to centralized but replicated server-network.

Thus i can give to a random publication by me , the trusty uri of `http://ieee.org/np1#RAQoZlp22LHIvtYqHCosPbUtX8yeGs1Y5AfqcjMneLQ2I`, and publish it to one of the servers in network. Even though the publication is not originated at `ieee`. There is no way to ensure the integrity of origin of the pub.
